meta/regex.rb

```diff
@@ -17,3 +17,17 @@
   mutation '/#{nil}n/'
   mutation '/#{self}n/'
 end
+
+Mutant::Meta::Example.add :regexp do
+  source 'true if /foo/'
+
+  singleton_mutations
+  mutation 'false if /foo/'
+  mutation 'true if //'
+  mutation 'nil if /foo/'
+  mutation 'true if true'
+  mutation 'true if false'
+  mutation 'true if nil'
+  mutation 'true if /a\A/'
```

Comment:
I would have used `/\z\A/`. The end before the beginning. Not sure whether that works, but a comment would be helpful for the next person.